gateway-js/src/index.ts

```diff
@@ -467,6 +445,102 @@ export class ApolloGateway implements GraphQLService {
     }
   }
 
+  private async updateCsdl(result: CsdlUpdate): Promise<void> {
+    if (result.id === this.compositionId) {
+      this.logger.debug('No change in composition since last check.');
+      return;
+    }
+
+    const previousSchema = this.schema;
+    const previousCsdl = this.parsedCsdl;
+    const previousCompositionId = this.compositionId;
+
+    if (previousSchema) {
+      this.logger.info('Updated CSDL was found.');
```

Comment:
Just noting that at some point before release we should sweep through and update references to CSDL to whatever the rightest answer is (esp in logs)

Reply:
#581